dex/pair/src/lib.rs

```diff
@@ -146,6 +146,8 @@ pub trait Pair<ContractReader>:
         self.execute_output_payments(&context);
         self.emit_add_liquidity_event(&context);
 
+        self.initial_liquidity_adder().clear();
```

Comment:
why do you need this? you just want to clear the storage ?

Reply:
I see - this needs to be empty on normal addLiquidity

Reply:
There is a check in the normal `addLiquidity` endpoint to not have a price discovery address set in order to add liquidity